SingularityService/src/test/java/com/hubspot/singularity/scheduler/SingularitySchedulerTest.java

```diff
@@ -510,7 +510,7 @@ public void testOneOffsDontMoveDuringDecomission() {
     requestResource.postRequest(bldr.build());
     deploy("d2");
 
-    requestResource.scheduleImmediately(requestId);
+    requestResource.scheduleImmediately(requestId, Optional.absent());
```

Comment:
would it have made sense to create a method here that didn't take the second parameter?